pkg/cmd/auth/setupgit/setupgit_test.go

```diff
@@ -59,6 +67,16 @@ func Test_setupGitRun(t *testing.T) {
 			expectedErr:    "You are not logged into the GitHub host \"foo\"\n",
 			expectedErrOut: "",
 		},
+		{
+			name: "add credential helper for unknown host",
```

Comment:
Please repeat this test with no hosts, it will currently fail but it should work.